src/main/java/com/epam/ta/reportportal/core/configs/RabbitMqConfiguration.java

```diff
@@ -108,8 +132,8 @@ public SimpleRabbitListenerContainerFactory rabbitListenerContainerFactory(@Auto
 		factory.setConnectionFactory(connectionFactory);
 		factory.setDefaultRequeueRejected(false);
 		factory.setMessageConverter(jsonMessageConverter());
-		factory.setConcurrentConsumers(3);
-		factory.setMaxConcurrentConsumers(10);
+		factory.setConcurrentConsumers(1);
```

Comment:
left from debugging, reverted.
it will be interesting to play this params on performance test. may be single consumer or small amount would be better, if two many retries happen with many parallel consumers. also max attempt faults could happen.